modules/installation-vsphere-infrastructure.adoc

```diff
@@ -18,7 +18,7 @@ You must install the {product-title} cluster on a VMware vSphere version 7.0 Upd
 {product-title} version {product-version} supports VMware vSphere version 8.0.
 ====
 
-You can host the VMware vSphere infrastructure on-premise or on a link:https://cloud.vmware.com/providers[VMware Cloud Verified provider] that meets the requirements outlined in the following table:
+You can host the VMware vSphere infrastructure on-premise or on a link:https://cloud.vmware.com/providers[VMware Cloud Verified provider] that meets the requirements outlined in the following tables:
```

Comment:
Moved note further up, as listing multiple notes does not lead to a good customer UX.